styles/styles.css

```diff
@@ -341,13 +341,13 @@ main > .section > .default-content-wrapper > p {
   left: 3vw;
```

Comment:
The content keeps shifting as the browser resizes. Please change this VH value to a static one.

Reply:
I suppose the content should shift a bit as the browser resizes, thats why leveraged vw. Sure, fixed that with a static value.

Reply:
Can you remove the "margin-bottom: 2vh" as well as the left: 3vw?

Reply:
margin-bottom: 2vh is removed. Left: 20px is already set.